app/models/validations/sales/soft_validations.rb

```diff
@@ -1,5 +1,5 @@
 module Validations::Sales::SoftValidations
-  ALLOWED_INCOME_RANGES = {
+  ALLOWED_INCOME_RANGES_SALES = {
```

Comment:
I wonder if the other one of these should be names `..._LETTINGS` or `..._SHARED` (depending which of those it is)